### ISSUE TITLE
[GIT] substr() expects parameter 1 to be string, null given in GitRevision.php:43

### ISSUE THREAD

**Issue by OlafvdSpek** [phpmyadmin/phpmyadmin#14762]
```
PHP message: PHP Fatal error:  Uncaught TypeError: substr() expects parameter 1 to be string, null given in /srv/phpMyAdmin-5.0+snapshot-english/libraries/classes/Display/GitRevision.php:43
Stack trace:
#0 /srv/phpMyAdmin-5.0+snapshot-english/libraries/classes/Display/GitRevision.php(43): substr(NULL, 0, 7)
#1 /srv/phpMyAdmin-5.0+snapshot-english/index.php(131): PhpMyAdmin\Display\GitRevision::display()
#2 {main}
  thrown in /srv/phpMyAdmin-5.0+snapshot-english/libraries/classes/Display/GitRevision.php on line 43
```

**Comment by williamdes** [comment]
Duplicate of #14635

**Comment by williamdes** [comment]
@OlafvdSpek Thank you for the report !
Can you send a detailed way of reproducing this issue ?
That would help me close #14635 and this one.

**Comment by OlafvdSpek** [comment]
Install snapshot, open pMA.

**Comment by williamdes** [comment]
@OlafvdSpek I can not reproduce this issue using the snapshot. :disappointed: 
Can you please send more instructions ?

**Comment by OlafvdSpek** [comment]
PHP 7.3?

**Comment by OlafvdSpek** [comment]
Caching maybe? I replaced the git version with the snapshot..

**Comment by williamdes** [comment]
@OlafvdSpek To reproduce
- Install using git
- `mv .git .gitold`

The issue is reproduced !

**Comment by williamdes** [comment]
Will be fixed as issue #14635
Thank you for helping @OlafvdSpek !
